lib/util/constants.js

```diff
@@ -6,7 +6,7 @@ const constants = {
   SYSTEM_ALIAS: '--system-alias',
   PASS_ARGS: '--pass-args',
   VERSION: '--version',
-  ROOT_DIR: '/usr/local/.squash',
+  ROOT_DIR: `${require('os').homedir()}/.local/bin`,
```

Comment:
I am not so sure about this.
Is it a good practice if we store all commands in `/.local/bin` ?
Or we can think of something else, so we don't have to create sh files we can do something else,
Like store the command in a json or something and executing it when we try to run it. 
I am not so sure how exactly we will do this, but I am thinking of something similar in this path.

Reply:
Sh files are the simpler way. Storing the commands in a json will require us to run javascript code in order to execute. The sh files in the PATH are already used by bash.
I think this can also help with multi-user environment, making the squashed commands user-wide and not requiring root access to modify files inside `/usr/local/`

Reply:
Ok. Let's keep it in /usr/local for v2.
We can modify it for further release if we need